src/server/pfs/server/driver.go

```diff
@@ -3441,16 +3443,17 @@ func (d *driver) forEachPutFile(pachClient *client.APIClient, server pfs.API_Put
 			})
 		}
 		if _, err := pw.Write(req.Value); err != nil {
-			return oneOff, repo, branch, err
+			return false, "", "", err
 		}
 	}
 	if pw != nil {
-		// This may pass io.EOF to CloseWithError but that's equivalent to simply calling Close()
+		// This may pass io.EOF to CloseWithError but that's equivalent to
+		// simply calling Close()
 		pw.CloseWithError(err) // can't error
 	}
 	if err != io.EOF {
-		return oneOff, repo, branch, err
+		return false, "", "", err
 	}
 	err = eg.Wait()
-	return oneOff, repo, branch, err
+	return false, "", "", err
```

Comment:
Should this return statement keep the old parameters? i.e. `return oneOff, repo, branch, nil`. Or is this an error  too?

Reply:
that would explain why my test is failing, heh